audio/tauonmb/tauonmb.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="tauonmb"
-VERSION="7.0.1"
+VERSION="7.1.1"
 HOMEPAGE="https://tauonmusicbox.rocks/"
-DOWNLOAD="https://github.com/Taiko2k/TauonMusicBox/archive/v7.0.1/TauonMusicBox-7.0.1.tar.gz"
-MD5SUM="6891a5020fa432104a328aedc7260d2e"
+DOWNLOAD="https://github.com/Taiko2k/TauonMusicBox/archive/v7.1.1/TauonMusicBox-7.1.1.tar.gz"
+MD5SUM="43dd7fc52e3b749254c35581b438db43"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
-REQUIRES="libopenmpt musicbrainzngs picard pylast python3-natsort python3-isounidecode python3-plexapi python3-pulsectl python3-pylyrics python3-pypresence python3-pysdl2 python3-tekore send2trash setproctitle websocket-client"
+REQUIRES="libopenmpt musicbrainzngs mutagen pylast python3-natsort python3-isounidecode python3-plexapi python3-pulsectl python3-pylyrics python3-pysdl2 python3-tekore send2trash setproctitle websocket-client"
```

Comment:
Some notes in the commit log about these changes would be good, e.g. "picard dep no longer needed, mutagen is new dep, etctera..."

Reply:
Oh, that's nothing,
Everytime I want to prepare new dependency, I have to discuss them, a lot..
mutagen is the dependency of tauonmb and picard, but I've mentioned picard and presence are optional, from now on,
because I didn't like the long list I saw after reinstalls!
Well sorry if I was rude.
I'll also try to be more respectful, and commit some changes

Reply:
Again, mutagen is not new dependency,
mutagen is common dependency of both.
Picard is for music tagging, internally
PyPresence is for Discord status view
(They're still dependencies)